### ISSUE TITLE
Member multipolygons are not highlighted when viewing site relation

### ISSUE THREAD

**Issue by neilguertin** [openstreetmap/openstreetmap-website#3841]
### URL

https://www.openstreetmap.org/relation/2048374#map=17/52.17360/0.14321

### How to reproduce the issue?

Search for "Addenbrooke's Hospital"
Click result for "Addenbrooke's Hospital (Main building)"
Click "Part of 1 relation Relation Addenbrooke's Hospital (2048374)"

### Screenshot(s) or anything else?

Addenbrooke's Hospital is the site relation highlighted here. The two larger buildings visible in this screenshot are also part of the site relation but are not highlighted because they are multipolygons.
![image](https://user-images.githubusercontent.com/8312387/207321523-2d02f698-790c-46d4-bd91-b8a50a650026.png)

**Comment by tomhughes** [comment]
Yes - it ignores ways and relations in nested relations (https://github.com/openstreetmap/leaflet-osm/blob/master/leaflet-osm.js#L266) and only processes nodes.

**Comment by mmd-osm** [comment]
„awaiting upstream“ -> where’s the corresponding upstream issue?

**Comment by AndrewKvalheim** [comment]
Related: #2772
